app/models/news_article.rb

```diff
@@ -8,6 +8,8 @@ class NewsArticle < Newsesque
   validates :news_article_type_id, presence: true
   validate :non_english_primary_locale_only_for_world_news_story
 
+  validate :policies_are_not_associated, unless: :can_be_related_to_policies?
```

Comment:
It does but it was messy trying to get the error message to make sense, particularly since I asked for a bit of content input on the correct text for the message.